css/app.css

```diff
@@ -0,0 +1,400 @@
+  align-items: flex-start;
+  justify-content: center;
+}
+
+.speaker_image .photo {
+  width: 120px;
+  height: 120px;
+}
+
+.info {
+  margin-left: 5px;
+}
+
+.name {
+  font-size: 1.5rem;
+  font-weight: 900;
+  color: #272a31;
+}
+
+.position {
+  color: #ec5242;
+  font-style: italic;
+  padding: 0.5rem 0;
+}
+
+.btn-default i {
+  font-size: 1rem;
+  color: #ec5242;
+  font-weight: 700;
+}
+
+#sponsors {
+  width: 100%;
+  padding: 2rem;
+  background-color: #272a31;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#sponsors h3 {
+  font-size: 1.5rem;
+  padding: 0 0 0.5rem;
+  color: #d3d3d3;
+}
+
+#sponsors hr {
+  width: 5%;
+  border: 1px solid #ec5242;
+}
+
+.sponsors {
+  padding-top: 2rem;
+  width: 80%;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  gap: 2rem;
+}
+
+.sponsors li {
+  list-style-type: none;
+}
+
+footer {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100px;
+}
+
+footer > div {
+  padding: 1.5rem 0;
+  width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+}
+
+.speakers > article:nth-child(n+3) {
+  display: none;
+}
+
+footer > div > a.logo {
+  display: block;
+}
+
+.copyright {
+  color: #606060;
+}
+
+.copyright > p:nth-child(1) {
+  font-size: 0.8rem;
+  color: #484848;
+}
+
+.copyright > p:nth-child(2) {
+  display: none;
+}
+
+#speakers button {
+  width: 90%;
+}
+
+@media all and (max-width: 350px) {
+  .greeting {
+    padding: 5rem 0 1rem;
+  }
+}
+
+@media all and (min-width: 768px) {
+  .toolbar {
+    display: flex;
+  }
+
+  .nav-bar {
+    height: 70px;
+    background-color: #fff;
+    -webkit-box-shadow: 0 5px 7px -3px rgba(166, 161, 166, 1);
+    -moz-box-shadow: 0 5px 7px -3px rgba(166, 161, 166, 1);
+    box-shadow: 0 5px 7px -3px rgba(166, 161, 166, 1);
+  }
+
+  .logo {
+    margin-top: 10px;
+    display: block;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .menu-open {
+    display: block;
+  }
+
+  .nav-menu {
+    display: flex;
+  }
+
+  .nav-menu li {
+    margin-left: 35px;
+    margin-bottom: 0 !important;
+  }
+
+  .description {
+    width: 70%;
+    margin-right: 20%;
+  }
+
+  .pavilions {
+    width: 90%;
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    align-items: center;
+    gap: 0.5rem;
+  }
+
+  .stall {
+    flex-direction: column;
+    display: flex;
+    align-items: center;
+    height: 100%;
+  }
+
+  .stall img {
+    height: 80px;
+    width: 80px;
+  }
+
+  .stall h3 {
+    width: 100%;
+  }
+
+  .stall p {
+    text-align: center;
+  }
+
+  .speakers {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  #speakers button {
+    display: none;
+  }
+
+  .sponsors {
+    width: 100%;
+  }
+
+  .sponsors img {
+    width: 150px;
+    height: auto;
+  }
+
+  footer {
+    height: 100px;
+  }
+
+  .afooter {
+    background: #2e2e2e;
+  }
```

Comment:
- **[OPTIONAL]** Kindly add color: #000 for a good contrast for a better UI 👇🏽 
- **[OPTIONAL]** In the about page for the desktop version there isn't the Sponsors section, please 🙏🏽 remove it to match the original design.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219695608-e808e02d-8b01-4b68-a4a8-f1575acc6c96.png) | ![image](https://user-images.githubusercontent.com/58642949/219695820-b294cfd2-42a7-403a-a1db-8f5512e5a462.png) |
| ![image](https://user-images.githubusercontent.com/58642949/219696275-0ef30b10-c555-4ad2-b07b-e436ab1ed1c4.png) | ![image](https://user-images.githubusercontent.com/58642949/219696351-b4f6a1bf-6efe-499c-b624-e9efba44b51b.png) |